go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/rootless-containers/proto v0.1.0
 	github.com/sirupsen/logrus v1.6.0 // indirect
+	github.com/stretchr/testify v1.6.1
```

Comment:
It appears this new dependency is missing from the `vendor` tree:

```
# github.com/opencontainers/umoci/oci/layer
package github.com/opencontainers/umoci/oci/layer (test)
	imports github.com/stretchr/testify/assert: cannot find package "github.com/stretchr/testify/assert" in any of:
	/home/travis/gopath/src/github.com/opencontainers/umoci/vendor/github.com/stretchr/testify/assert (vendor tree)
	/home/travis/.gimme/versions/go1.14.4.linux.amd64/src/github.com/stretchr/testify/assert (from $GOROOT)
	/home/travis/gopath/src/github.com/stretchr/testify/assert (from $GOPATH)
FAIL	github.com/opencontainers/umoci/oci/layer [setup failed]
```

Reply:
Yeah, and I couldn't figure out how to actually make vendor add/ it. But upon looking at it again, I had vendor/ in my global gitignore :). Anyway, let's see if this patch works.